src/components/home/SeeMoreButton.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
+import React, { useState } from 'react';
+import {jsx, SxStyleProp} from 'theme-ui';
+
+
+
+interface SeeMoreProps {
+  expand: () => void;
+  style?: SxStyleProp;
+}
+
+/**
+ * A button that disappears if clicked (on mobile view).
+ * 
+ * You can pass in an `expand` function that executes when the button is clicked.
```

Comment:
Forgot how JS likes it but should you define what kind of function should belong here? Or does JSDoc provide you the type signature?

Even if it does I think there is merit in describing what kind of function (in terms of return type and parameters) should be passed.